content/_index.md

```diff
@@ -1,8 +1,77 @@
 +++
-title = "Snowplow Accelerator"
-menuTitle="Introduction"
+title = "Introduction"
+menuTitle = "Introduction"
+pre = "<i class='fas fa-rocket'></i> "
 chapter = false
 weight = 1
 +++
 
-This is an example of an Accelerator Introduction
\ No newline at end of file
+!['logo-banner'](images/logo_banner_2.png)
+
+Welcome to the **Fractribution** accelerator. This is a tutorial for a Snowplow accelerator that demonstrates how to run fractional attribution analysis on Snowplow data, coined 'Fractribution'. Fractional attribution allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as the return on advertising spend (ROAS) if you have spend data for each marketing channel. Once finished, you will be able to build a deeper understanding of customer behaviour on your website and use your data to influence business decisions.
+
+Here you will learn to:
+
+* Model and Visualise Snowplow data
+  - using the [fractribution](https://hub.getdbt.com/snowplow/fractribution/latest/) dbt package and Python script
+  - using our sample data for Snowflake (no need to have a working pipeline)
+* Set-up Snowplow Tracking and Enrichment for fractribution
+* Apply what you have learned on your own pipeline to gain insights
+***
+
+#### Who is this guide for?
+
+- Data practitioners who would like to get familiar with further options for modelling Snowplow data.
```

Comment:
Done. In the same vein, I followed the convention from the web analytics accelerator package and used 'visualisation' throughout, instead of the American 'visualization'. Should I change these to American spelling, or keep consistent with the other accelerator? Similar to set up vs set-up, I read that set up is more commonly American, with set-up more commonly British. 
They certainly don't make it easy!

Reply:
Thanks for doing all these changes. It is quite a challenge to keep it all consistent with the dual languages. 
So, think consistency is key here now:
Keep "Visualisation"
Keep "set-up"

Reply:
Okay will do, thanks.